### PR TITLE
Prep for v1.13.0

### PR DESCRIPTION
**Pull request by odow**
Wait for:

 - [x] #2101
 - [x] https://github.com/jump-dev/MathOptInterface.jl/pull/2103
 - [x] #2105 
 - [x] #2106 
 - [x] #2107
 - [x] #2109

### CODE REVIEW

**Comment by blegat** [comment]
Let's see what https://github.com/jump-dev/SolverTests/actions/runs/4293473998 gives

**Comment by odow** [comment]
Started a new SolverTests: https://github.com/jump-dev/SolverTests/actions/runs/4297638598

**Comment by odow** [comment]
This time perhaps?  https://github.com/jump-dev/SolverTests/actions/runs/4299104395

File: docs/src/changelog.md [hunk 1]
```diff
@@ -7,6 +7,30 @@ CurrentModule = MathOptInterface
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.13.0 (February 28, 2022)
+
+### Added
+
+ - Added [`Bridges.Constraint.NumberConversionBridge`](@ref) (#2091)
+ - Added [`Parameter`](@ref) set (#2095) (#2105) (#2106)
```
Comment (odow) [inline docs/src/changelog.md:15]:
```suggestion
 - Added [`Parameter`](@ref) set (#2095) (#2105) (#2106) (#2109)
```

**Comment by odow** [comment]
Looks like we're good to go. SolverTest failures are unrelated.

**Pull request merged**